### ISSUE TITLE
模型训练时，outputs = model(drr) 输出size有误？？？

### ISSUE THREAD

**Issue by akonepoint** [LijunRio/Xrays_CT#2]
Q. 模型训练时，运行到 loss = criterion(outputs, ct) 提示 target size 不等于 input size ？？

UserWarning: Using a target size (torch.Size([1, 128, 128, 128])) that is different to the input size (torch.Size([16, 128, 128, 128])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.

A.1. 回到 outputs = model(drr)， 发现输出shape 是[16，128，128，128] 与 target [1,128,128,128] 不一致，以至于后续测试模型 squeeze 时，报错，无法后续执行。

2. 追溯到 net.py，  在 transform module 的 trans_features 已生成 16那维。
![image](https://user-images.githubusercontent.com/82929985/115508731-3d38ab80-a2b0-11eb-81f6-3d0e7922e847.png)

3. 请问该如何消除源码设为 -1的那维？（代码没有变动）

**Comment by akonepoint** [comment]
请问： drr 输入size 是 [1,1,512,512],还是 [1,1,128,128]?

**Comment by BrightenZL** [comment]
@akonepoint 请问你有解决吗？

**Comment by akonepoint** [comment]
修改输入数据的 drr size， 原 512*512， 改为 128* 128 。 代码可运行。但好像最后生成的是全黑图像。。

> @akonepoint 请问你有解决吗？

**Comment by quocbao2772004** [comment]
@akonepoint 
Hi bro, have you fixed it, I tried resize images from 512_512 to 128_128 but the result I got was all black
